website/src/components/DataUseTerms/DataUseTermsSelector.tsx

```diff
@@ -1,25 +1,82 @@
-import { type FC } from 'react';
+import { Datepicker } from 'flowbite-react';
+import { DateTime } from 'luxon';
+import { useState, type FC } from 'react';
 
+import { DateChangeModal, datePickerTheme } from './DateChangeModal.tsx';
+import { getClientLogger } from '../../clientLogger.ts';
 import { routes } from '../../routes/routes.ts';
-import { type DataUseTermsType, openDataUseTermsType, restrictedDataUseTermsType } from '../../types/backend.ts';
+import {
+    type DataUseTermsType,
+    openDataUseTermsType,
```

Comment:
observation (not requesting code change atm): we should have named this `openDataUseTermsOption` - calling it `type` makes me think its a TypeScript type and confused me

Reply:
Oh, actually I think the same! I think naming things accurately is great. done: [`9b87eed` (#3322)](https://github.com/loculus-project/loculus/pull/3322/commits/9b87eedccbf344b50bf48154719d3127cff1e426)

Reply:
Thanks!